nvi-rest/src/main/java/no/sikt/nva/nvi/rest/upsert/NviStatusRequest.java

```diff
@@ -11,6 +12,12 @@ public record NviStatusRequest(UUID candidateId,
                                ApprovalStatus status,
                                String reason) {
 
+    public void validate() {
+        if (isNull(candidateId) || isNull(institutionId) || isNull(status)) {
+            throw new IllegalArgumentException("Missing required fields");
```

Comment:
Adding this as a sanity check, because I assume it should not be possible to update the status of an NVI candidate without specifying these fields.